systemtest/src/test/resources/oauth2/prepare_keycloak_operator.sh

```diff
@@ -6,10 +6,66 @@ KEYCLOAK_INSTANCE_NAMESPACE=$3
 
 SCRIPT_PATH=$(dirname "${BASH_SOURCE[0]}")
 
+function create_keycloak_cr() {
+cat <<EOF > /tmp/keycloak_pg_example.yaml
+apiVersion: keycloak.org/v1alpha1
+kind: Keycloak
+metadata:
+  name: example-keycloak
+  labels:
+    app: sso
+spec:
+  instances: 1
+  extensions:
+    - https://github.com/aerogear/keycloak-metrics-spi/releases/download/2.5.3/keycloak-metrics-spi-2.5.3.jar
```

Comment:
Will we need those metrics (or for what it is used)?

Reply:
By default it was used in example keycloak. (version 1.4.0 or 1.0.4 idk). It was part of issue why KC did not work (too old plugin). I tried not to use it, but it seems like there was some default (aforementioned 1.4.0) which it failed installation and startup of KC. :/

Reply:
So without some metrics plugin, the Keycloak will not start?

Reply:
I will give it a shot again tomorrow, but I think it will use that crappy old 1.4.0 version (by default).

Reply:
Works w/o metrics plugin as well. Seems like there was some old remnant of CR settings, that I have been playing with and failed at that time. Better not to have it there then. ;)